Selenium/Ex8.rb

```diff
@@ -0,0 +1,31 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path = 'C:\Users\adity\Desktop\Drivers\chromedriver-win64\chromedriver-win64\chromedriver.exe'
+driver = Selenium::WebDriver.for :chrome
+
+
+driver.get 'https://www.globalsqa.com/demo-site/sliders/#Steps'
+
+
+slider_handle = driver.find_element(:xpath, ``)
```

Comment:
The slider is in a frame, you need to switch to the frame before getting the slider element